inst/examples/translation.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Thoughts:

1. As I wrote above, I'd consider having separate translation files for each language.
2. Why JSON? this format seems overly verbose and hard to work with for non-technical people. Maybe we could have some simple format like CSV? for example `message_id;translation` (where message_id is the same as text in the fallback language)  -- WDYT?
3. btw I now have a thought that this format could be switchable... you could switch between JSON, CSV, Excel, or your own format... but this is probably for the future :) we now need some sane default that will be easy for people to work with, or convert to from other formats if they need to